bre/bre.py

```diff
@@ -1,2 +1,79 @@
+import csv
+
+
+class Story:
+    def __init__(self, pages, current_page):
+        self.pages = pages
+        self.current_page = current_page
```

Comment:
I'm tempted to change this to current_page_id  to make it clear that it's the page number rather than the page object that goes here. 

But we can change it in a future PR